generic/_generic.reset.scss

```diff
@@ -3,20 +3,63 @@
    ========================================================================== */
 
 /**
- * A very simple reset that sits on top of Normalize.css.
+ * Add the correct display in IE.
  */
 
-body,
-h1, h2, h3, h4, h5, h6,
-blockquote, p, pre,
-dl, dd, ol, ul,
+main {
+  display: block;
+}
```

Comment:
If we don’t want to support any IE anymore with version 7, I guess we don’t need this reset.

Reply:
Agree

Reply:
Scratch that. No version of IE (not even IE11) does support the `<main>` element. So we better leave this in.